internal/command/deploy/machines_tigrisstatics.go

```diff
@@ -0,0 +1,617 @@
+		ErrorCaptureCallback: nil,
+		OverrideRegion:       md.appConfig.PrimaryRegion,
+		OverrideName:         &md.tigrisStatics.bucket,
+	}
+	params.Options["website"] = map[string]interface{}{
+		"domain_name": "",
+	}
+	params.Options["accelerate"] = false
+	// TODO(allison): Make sure we still need this when virtual services drop :)
+	params.Options["public"] = true
+
+	ext, err := extensions.ProvisionExtension(ctx, params)
+	if err != nil {
+		return err
+	}
+
+	secrets := ext.Data.Environment.(map[string]interface{})
+
+	tokenizedKey, err := md.staticsTokenizeTigrisSecrets(ctx, org, secrets, tokenizerSealKey)
+
+	// TODO(allison): Temporary, while working on the POC for Tokenizer
+	return os.WriteFile("tokenized_key", ([]byte)(tokenizedKey), 0644)
+
+	return err
+}
+
+func (md *machineDeployment) staticsTokenizeTigrisSecrets(
+	ctx context.Context,
+	org *fly.Organization,
+	secrets map[string]interface{},
+	tokenizerSealKey string,
+) (string, error) {
+
+	client := flyutil.ClientFromContext(ctx)
+
+	orgId, err := strconv.ParseUint(org.InternalNumericID, 10, 64)
+	if err != nil {
+		return "", fmt.Errorf("failed to decode org ID for %s: %w", org.Slug, err)
+	}
+
+	// TODO(allison): We should just pass the whole app to deploy instead of re-grabbing it here
+	app, err := client.GetApp(ctx, md.app.Name)
+	if err != nil {
+		return "", err
+	}
+	appId := uint64(app.InternalNumericID)
+
+	type sigv4Processor struct {
+		AccessKey string `json:"access_key"`
+		SecretKey string `json:"secret_key"`
+	}
+	type flyioMacaroonAuthConfig struct {
+		Access flyio.Access `json:"access"`
+	}
+	type tokenizeInput struct {
+		Sigv4Processor          sigv4Processor          `json:"sigv4_processor"`
+		FlyioMacaroonAuthConfig flyioMacaroonAuthConfig `json:"flyio_macaroon_auth"`
+		AllowedHosts            []string                `json:"allowed_hosts"`
+	}
+
+	// TODO(allison): How do we handle moving an app between orgs?
+	//                We're locking this token behind a hard dependency on the App ID and Org ID, but I don't believe
+	//                *either* of these stay static when moving from one org to another.
+	//
+	input := tokenizeInput{
+		Sigv4Processor: sigv4Processor{
+			AccessKey: secrets["AWS_ACCESS_KEY_ID"].(string),
+			SecretKey: secrets["AWS_SECRET_ACCESS_KEY"].(string),
+		},
+		FlyioMacaroonAuthConfig: flyioMacaroonAuthConfig{
+			Access: flyio.Access{
+				Action: resset.ActionWrite,
+				OrgID:  &orgId,
+				AppID:  &appId,
+			},
+		},
+		AllowedHosts: []string{tigrisHostname},
+	}
+
+	inputJson, err := json.Marshal(input)
+	if err != nil {
+		return "", err
+	}
+
+	pubBytes, err := hex.DecodeString(tokenizerSealKey)
+	if err != nil {
+		return "", err
+	}
+	if len(pubBytes) != 32 {
+		return "", fmt.Errorf("bad public key size: %d", len(pubBytes))
+	}
+
+	sct, err := box.SealAnonymous(nil, inputJson, (*[32]byte)(pubBytes), nil)
+	if err != nil {
+		return "", err
+	}
+
+	return base64.StdEncoding.EncodeToString(sct), nil
```

Comment:
I'm curious why you aren't using the tokenizer library for doing all this? You should be able to do something like

```go
secret := &tokenizer.Secret{
  AuthConfig: tokenizer.NewFlyioMacaroonAuthConfig(....)
  ProcessorConfig: &tokenizer.Sigv4ProcessorConfig{...}
}

sealed, err := secret.Seal(sealKey)
```

If you want to use the version of tokenizer from your branch, you can run `go get github.com/superfly/tokenizer@feat/sigv4`